### PR TITLE
Adding plotting script for ArborX query order

### PR DESCRIPTION
**Pull request by aprokop**
Co-authored-by: Damien L-G <dalg24@gmail.com>

Not sure where to put it, so it's in `scripts` for now. I would like to carry it with the repo so it could be used anytime without access to Jupyter.

There are some quality of life improvements compared to original @dalg24's version:
- Command line arguments support
- Automatic calculation of the number of files to parse
- The ability to either display or save to a file

Right now, there is no test for it, but I could add it if considered useful. For example, I could add a test for visualization driver to dump data, and run the script just to make sure it runs.

### CODE REVIEW

File: scripts/arborx_query_sort.py [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+    algo = options['--algo']
+    display = options['--display']
+    prefix = options['--prefix']
+    output_file = options['--output-file']
+
+    matrix = {}
+
+    n = int(subprocess.check_output('ls -1 ' + prefix + '*' + algo + '* '
+                                    '| wc -l', shell=True))
```
Comment (dalg24) [inline scripts/arborx_query_sort.py]: Nice

File: scripts/arborx_query_sort.py [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+    print(n)
```
Comment (dalg24) [inline scripts/arborx_query_sort.py]: Might want to remove that print statement.

**Comment by dalg24** [comment]
plot
For the record, in the dev/ci container I had to do `apt install python3-pip` and `pip3 install -r requirements.txt`

**Comment by aprokop** [comment]
@dalg24 I assume that in the container one is able to run only saving to file, and not in display mode, correct?

**Comment by dalg24** [comment]
> @dalg24 I assume that in the container one is able to run only saving to file, and not in display mode, correct?

I only saved a the plot.  In principle I think it is possible if you run your container with `-e DISPLAY=$DISPLAY -v /tmp/.X11-unix:/tmp/.X11-unix` or similar but I didn't try cause I connected remotely and do not really want to bother with X forwarding.

**Pull request merged**